state/state.go

```diff
@@ -250,22 +248,22 @@ func (st *HelmState) prepareSyncReleases(helm helmexec.Interface, additionalValu
 
 				flags, flagsErr := st.flagsForUpgrade(helm, release, workerIndex)
 				if flagsErr != nil {
-					results <- syncPrepareResult{errors: []*ReleaseError{&ReleaseError{release, flagsErr}}}
+					results <- syncPrepareResult{errors: []*ReleaseError{&ReleaseError{release, flagsErr, 1}}}
 					continue
 				}
 
 				errs := []*ReleaseError{}
 				for _, value := range additionalValues {
 					valfile, err := filepath.Abs(value)
 					if err != nil {
-						errs = append(errs, &ReleaseError{release, err})
+						errs = append(errs, &ReleaseError{release, err, 1})
```

Comment:
that would be great if 1 would be a constant with a name so that we know what is this parameter, it would make the code easier to read.

Reply:
Good point! I'd introduce three constants, `ExitStatusSucess`, `ExitStatusFailure`, `ExitStatusDiff` for `0`, `1`, `2` respectively.

Reply:
I ended up just avoid using magic numbers whereas possible. Hope it makes sense now!